src/toolbars/SpectralSelectionBar.cpp

```diff
@@ -179,36 +179,39 @@ void SpectralSelectionBar::Populate()
    {
       auto subSizer = std::make_unique<wxBoxSizer>(wxHORIZONTAL);
 
-      mCenterCtrl = safenew NumericTextCtrl(
+      mCenterCtrl = safenew NumericTextCtrl(FormatterContext::ProjectContext(mProject),
```

Comment:
Might as well do this for the spectral selection bar too, though in fact sample rate is never used when formatting frequencies or bandwidths.